javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $body = document.querySelector('.body-white');
+var $circle = document.querySelector('.yellow-circle');
+
+function handleColorChange(event) {
+  if ($circle.className === 'yellow-circle') {
```

Comment:
This is making your JavaScript dependent on data from the DOm to know what to do. As we discussed in our `dom-manipulation` Q&A, that is not ideal. Instead, you should make a variable in your javascript that represents whether the light is currently on or not, and give it a boolean value.